docs/DeveloperGuide.md

```diff
@@ -1,34 +1,662 @@
+The `CommandEventDel` class in the same package handles deleting of a event. It deletes an `Event` instance according to the index provided by `userInput` from the `EventList`.  
+They implement the following operations:  
+* `CommandEventAdd#execute()` - Adds a new event into the `EventList` according to `userInput`.  
+* `CommandEventDel#execute()` - Deletes an event from `EventList` or deletes all the events in the list. 
+To delete a particular event, enter the index of the event.
+To delete all the events in the list, enter `all` instead of the index of the event.
+
+Given below is an example usage scenario and how add/delete event function behaves at each step.  
+
+Step 1. The user launches the application for the first time.   
+
+Step 2. The user executes `event addEvent /n arduino course /d 2020-12-30 /t 8pm` command to add a new event with the name "arduino course", 
+the date of the event "2020-12-30" and the time "8pm" into event list. 
+The `event addEvent` command calls `CommandEventAdd#execute()`, then `EventList` will add a new `Event` with event name as `iphone12`, date as `2020-12030` and time as `8pm`.  
+
+Step 3. The user executes `event delEvent 1` command to delete the 1st event in the event list. The `event delEvent`
+command calls `CommandEventDel#execute()`, causing the `Event` at index 1 to be removed from `EventList`.  
+
+The sequence diagram for adding an event is as shown below:
+
+![CommandAddEventAttendance](EventDiagram/SequenceDiagram/CommandAddEventAttendance.png)
+
+The sequence diagram for deleting a particular event or all events is as shown below:
+
+![CommandDelEventAttendance](EventDiagram/SequenceDiagram/CommandAddEventAttendance.png)
+
+**3.4.7. Listing Events** `CommandEventList`
+
+3.4.7.1 Current implementation
+The `CommandEventList` class in `seedu.duke.event` handles listing all the events in `EventList`.
+
+It implements the following operation:  
+* `CommandEventList#execute()` - Lists all `Event` in `EventList`.  
+
+Given below is an example usage scenario and how the program list the events.  
+
+Step 1. After some `Event addEvent` commands, the user has created a `EventList` with some `Event`. Assuming there are 2 events in the list.
+The first `Event` has the name arduino course on 30 December 2020 at 8pm and the second `Event` has the name Autodesk course on 25 May 2021 from 10-12pm.
+
+![](EventDiagram/2Step1.png)
+
+Step 2.The user executes `event listEvent` command to list the `EventList`. The `event listEvent` command calls 
+`CommandEventList#execute()`, then every `Event` in `EventList` will be printed out. Nothing will be changed in `EventList`.  
+
+![](EventDiagram/Step2.png)
+
+The sequence diagram for listing events is as shown below:
+
+![](EventDiagram/SequenceDiagram/CommandEventList.png)
+
+### 3.5. HR
+The diagram below shows the architecture for HR feature.<br/>
+
+![](hrDiagramPic/HrArchitecture.png)
```

Comment:
<img width="843" alt="Screenshot 2020-10-30 at 12 31 08" src="https://user-images.githubusercontent.com/59906930/97660211-d7f7c080-1aab-11eb-9f3f-474222baee5c.png">

Again, diagram here looks too complicated and small, maybe it could be broken down to smaller pieces!